langlib/langlib-test/src/test/java/org/ballerinalang/langlib/test/LangLibRegexpTest.java

```diff
@@ -69,4 +71,26 @@ public Object[] testRegexLangLibFunctions() {
                 "testSplit"
         };
     }
+
+    @Test(dataProvider = "negativeRegexpFindIndexProvider")
+    public void testNegativeRegexp(String functionName, int startIndex, int length) {
+        Object returns = BRunUtil.invoke(negativeTests, functionName);
+        Assert.assertEquals(returns.toString(),
```

Comment:
According to our coding styles, don't we need to do this assertion in Ballerina files, wdyt?

Reply:
Since this causes a runtime error, it would not be possible to have this in the Ballerina source code. The goal here is to capture the runtime error that occurs once the Ballerina function is invoked.